app/Broadcasts/LogSmash/App/LogAppImageStoreRequestBroadcast.php

```diff
@@ -0,0 +1,85 @@
+<?php
```

Comment:
Sorry to be a pedant, but App really doesn't describe this at all.

Can it not be called "PhotoApp" or "ImageApp" or something? There will be a day when we write another App so they should be differentiated.

Reply:
I had originally thought of that, but the current app is being extended to cover a number of different things. This app is currently referred to as the Apex App, so I believe subsequent apps would be named accordingly like Apex Pin The Tail On The Donkey App. In this case the current app will always be the main app and others will be children apps.